.travis.yml

```diff
@@ -14,11 +14,16 @@ install:
   - pip install -q Django==$DJANGO
   - pip install -q pytz==2016.4
   - pip install -q python-social-auth==0.2.19
+  - pip install selenium
 
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y scilab
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
 
 # command to run tests
 script:
   - python manage.py test -v 2 yaksh
+  - python manage.py test -v 2 yaksh.tests.load_test
```

Comment:
The load_test.py uses a different type of TestCase class called LiveServerTestCase. The unit tests use the usual TestCase class.
The LiveServerTestCase class deals with Tear down a little differently - It only removes the rows of the table and does not reset the Primary keys. Because of this the subsequent model test cases fail.
Everything runs fine if both model test cases and selenium load test are run seperately.

The situation that I faced has been described in the question below. I tried a few solution but nothing seems to fix the issue
http://stackoverflow.com/questions/32998166/with-py-test-database-is-not-reset-after-liveservertestcase

I have posted this to the django user google group/mailing list, but the message seems to be pending approval.